src/lib/Microsoft.Health.Common/Microsoft.Health.Common.csproj

```diff
@@ -18,19 +18,18 @@
   <ItemGroup>
     <PackageReference Include="Azure.Storage.Blobs" Version="12.8.0" />
     <PackageReference Include="Ensure.That" Version="9.2.0" />
-    <PackageReference Include="Microsoft.Extensions.Options" Version="5.0.0" />

```

Comment:
This was the specific package upgrade in the previous update causing the function app to crash on Azure App Services.